package ExceptionDemo;
public class ExceptionDemo01{
	public static void main(String args[]){
		int i = 10 ;
		int j = 0 ;
		System.out.println("========== 计算开始 ===========") ;
		System.out.println("计算结果：" + i / j) ;
		System.out.println("========== 计算结束 ===========") ;
	}
};